tests/scripts/run_embedded_app.py

```diff
@@ -1,5 +1,48 @@
 """Launch embedded instance."""
+import sys
+
 import ansys.mechanical.core as pymechanical
 
-# Launch embedded instance of app
-app = pymechanical.App()
+
+def launch_app(appdata_option):
+    """Launch embedded instance of app."""
+    if appdata_option == "True":
+        app = pymechanical.App(private_appdata=appdata_option)
+    else:
+        app = pymechanical.App()
+    return app
+
+
+def set_false(appdata_option):
```

Comment:
I'd rather rename to `set_showtriad(appdata_option, value)` and remove `reset_showtriad` ?

Reply:
That makes sense, I like that better as well